docker-compose.yml

```diff
@@ -63,6 +63,11 @@ services:
     deploy:
       restart_policy:
         condition: any
+    configs:
+      - source: mis_config
```

Comment:
✏️ Should
```yaml
solr:
  host: TODO
  port: 9983
```
be removed from /multi-int-store/src/main/resources/application.yml?